src/sentry/seer/utils.py

```diff
@@ -183,4 +183,22 @@ def get_similar_issues_embeddings(
         )
         return []
 
-    return response_data.get("responses") or []
+    rv = []
+
+    for raw_similar_issue_data in response_data.get("responses") or []:
+        try:
+            rv.append(
+                SeerSimilarIssueData.from_raw(
+                    similar_issues_request["project_id"], raw_similar_issue_data
+                )
+            )
+        except (IncompleteSeerDataError, SimilarGroupNotFoundError) as err:
+            logger.exception(
```

Comment:
should we add some tests for these exceptions and logger?

Reply:
That's the part that codecov was complaining about. I realized there was a bug in my factory method (which I've gone back and fixed) which is why those lines weren't running and codecov was showing them as uncovered. So score one for codecov in this case!

As for testing the logging - I was torn about whether or not that was overkill, but I can.